ui_framework/components/table/landing_page_table/landing_page_tool_bar.js

```diff
@@ -8,28 +8,41 @@ import {
 export function LandingPageToolBar({
     filter,
     onFilter,
-    pagerState,
+    hasPreviousPage,
+    hasNextPage,
     onNextPage,
     onPreviousPage,
-    actionButtons }) {
+    actionButtons,
+    startNumber,
+    endNumber,
+    totalItems,
+}) {
   const toolBarSections = [
     <KuiToolBarSearchBox filter={filter} onFilter={onFilter}/>,
-    actionButtons
+    actionButtons,
+    <KuiToolBarPager
+      startNumber={startNumber}
+      endNumber={endNumber}
+      totalItems={totalItems}
+      hasPreviousPage={hasPreviousPage}
+      hasNextPage={hasNextPage}
+      onNextPage={onNextPage}
+      onPreviousPage={onPreviousPage}
```

Comment:
We're passing so many props down to the component tree, I wonder if there's some refactoring that needs to be done here. This would be something I'd probably want to tackle in a subsequent iteration.